code/modules/mob/living/carbon/human/emote_vr.dm

```diff
@@ -23,14 +23,37 @@
 		if("mlem")
 			message = "mlems [get_visible_gender() == MALE ? "his" : get_visible_gender() == FEMALE ? "her" : "their"] tongue up over [get_visible_gender() == MALE ? "his" : get_visible_gender() == FEMALE ? "her" : "their"] nose. Mlem."
 			m_type = 1
+		if("blep")
+			message = "bleps [get_visible_gender() == MALE ? "his" : get_visible_gender() == FEMALE ? "her" : "their"] tongue out. Blep."
+			m_type = 1
 		if("awoo")
+			m_type = 2
+			message = "lets out an awoo."
+			playsound(loc, 'sound/voice/long_awoo.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
+		if("awoo2")
 			m_type = 2
 			message = "lets out an awoo."
 			playsound(loc, 'sound/voice/awoo.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
```

Comment:
If you are going to add a new awoo sound, add that one as awoo2.   The current awoo sound should stay as-is.
```suggestion
			m_type = 2
			message = "lets out an awoo."
			playsound(loc, 'sound/voice/awoo.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
		if("awoo2")
			m_type = 2
			message = "lets out an awoo."
			playsound(loc, 'sound/voice/long_awoo.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
```